Gemfile

```diff
@@ -13,9 +13,9 @@ gem "optimist"
 gem "prometheus_exporter", "~> 0.4.5"
 gem "rake", ">= 12.3.3"
 gem "rest-client", "~>2.0"
-gem "sources-api-client", "~> 1.0"
+gem "sources-api-client", "~> 3.0"
 gem "topological_inventory-ingress_api-client", "~> 1.0.2"
-gem "topological_inventory-providers-common", "~> 1.0.2"
+gem "topological_inventory-providers-common", "~> 1.0.5"
```

Comment:
Can we change this to `~> 1.0`?

Reply:
we can change it only to `'~> 1.0', '>= 1.0.5'` so it doesn't worth